cli/server/server.go

```diff
@@ -183,8 +198,11 @@ func dumpDB(ctx *cli.Context) error {
 		}
 	}
 	chain.Close()
+	prometheus.ShutDown()
+	pprof.ShutDown()
```

Comment:
There has to be some symmetry, I think. If we start A, B and C then we should stop C, B and A.

Reply:
fixed!